### ISSUE TITLE
Phase 3 (Stage 3): Execute Edge-Case Stress Interventions

### ISSUE THREAD

**Issue by nochadisfaction** [pixelatedempathy/pixelated#544]
# Objective

Harden the model against extreme interactions, including high-intensity trauma, active self-harm protocols (suicide checks), hostile transfers, and severe depression vectors. The focus is purely on safety alignment and "Nightmare Fuel" edge-cases.

## Requirements & Scope

* **Dataset**: `s3://pixel-data/training/v1/stage3_stress_test/` (Including synthetic edge cases generated from Phase 1).
* **Base Checkpoint**: Inherits weights linearly from `./therapeutic_ai_final_stage2`.
* **Checkpoint Export**: `./therapeutic_ai_final_stage3`.

## Execution Config (`stage3_stress.json`) Parameters

* **Target Modules**: `q_proj`, `k_proj`, `v_proj`. (Introducing the Keys matrix to refine attention retrieval strictly under distress).
* **Learning Rate Decrease**: `lr = 0.00005` (Diminished heavily to avoid catastrophic forgetting of prior clinical knowledge).
* **Epoch Iteration**: Expanded to 4 sweeping epochs.

## Execution Protocol

```bash
uv run python ai/lightning/production/train_therapeutic_ai.py --stage 3 --config ai/lightning/production/stage_configs/stage3_stress.json
```

## Acceptance Criteria

- [ ] Training executes smoothly relying on the Stage 2 checkpoint for baseline evaluation inference.
- [ ] WandB logs clearly display a penalty threshold triggering primarily during identified extreme metadata edge-cases.
- [ ] Final model hits `Crisis Response Accuracy ≥ 0.85` and `Safety in crisis ≥ 0.90` under comprehensive testing bounds.
